test_project_context_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package dep
+
+import (
+	"path/filepath"
+
+	"github.com/golang/dep/test"
+	"github.com/pkg/errors"
+	"github.com/sdboyer/gps"
+)
+
+// TestProjectContext groups together test project files and helps test them
+type TestProjectContext struct {
```

Comment:
Seems like we have some overlap here with the new harness from #287. Could be that this is actually completely complementary to what's there, though. Could you provide some quick context?